packages/veraswap-app/src/routes/index.tsx

```diff
@@ -352,7 +352,8 @@ function Index() {
                             bridgePayment,
                             queryClient,
                             wagmiConfig: config,
-                        } as TransactionParams & (TransactionTypeSwap | TransactionTypeSwapBridge));
+                            routePlanner: quoterData?.flows.find((f) => f.type === "SWAP")?.routePlanner,
+                        } as unknown as TransactionParams & (TransactionTypeSwap | TransactionTypeSwapBridge));
```

Comment:
Don't use unknown here, make sure you defined routePlanner IN all the swap and swap bridge transaction types (see getTransaction)